contracts/src/v0.8/dev/ocr2dr/OCR2DR.sol

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import {CBORChainlink} from "../../../v0.7/vendor/CBORChainlink.sol";
+import {BufferChainlink} from "../../../v0.7/vendor/BufferChainlink.sol";
+
+/**
+ * @title Library for OCR2 Direct Request functions
+ */
+library OCR2DR {
+  uint256 internal constant defaultBufferSize = 256; // solhint-disable-line const-name-snakecase
+
+  using CBORChainlink for BufferChainlink.buffer;
+
+  enum CodeLocation {
+    Inline
+  }
+
+  enum CodeLanguage {
+    JavaScript
+  }
+
+  enum HttpVerb {
+    Get
+  }
+
+  struct HttpHeader {
+    string key;
+    string value;
+  }
+
+  struct HttpQuery {
+    HttpVerb verb;
+    string url;
+    HttpHeader[] headers;
+  }
+
+  struct Request {
```

Comment:
I was thinking about "DirectRequest" but I wanted to make a clear distinction between the existing DR feature and this OCR2DR feature.
Because this `Request` name is declared within `OCR2DR` library, a user will need to access it as `OCR2DR.Request` which is clear enough, no?